sensors_script/device_availability_monitor.py

```diff
@@ -42,7 +42,7 @@ def get_log_tail():
     read the last 10 lines of upflux and dpkg log file 
     """
     try:
-        result_dpkg = subprocess.run(['tail', '-n', '10', DPKG_LOG_PATH], stdout=subprocess.PIPE, text=True)
```

Comment:
Why did you change it from viewing the last 10 lines to the last 1 line?

Reply:
in the presentation I'm planning to show the error by injecting an error to the log file and then installing something so that the most recent is not an error log. if it checks for the last 10 lines it wont turn green when the most recent log is successful unless i installed the thing 10 times
.

Reply:
Okay